src/services/itemLogin/test/index.test.ts

```diff
@@ -239,6 +262,7 @@ describe('Item Login Tests', () => {
         const result = res.json();
         expect(result.id).toEqual(itemLoginSchema.id);
         expect(result.type).toEqual(itemLoginSchema.type);
+        expect(result.state).toEqual(itemLoginSchema.state);
         expect(result.item.id).toEqual(itemLoginSchema.item.id);
       });
 
```

Comment:
Add test to check you can still get item login schema even if it's disabled/frozen.